api/src/services/export/user-devices.js

```diff
@@ -0,0 +1,37 @@
+const bowser = require('bowser');
+
+const logger = require('../../logger');
+const db = require('../../db');
+
+const getBrowser = (userAgent) => {
+  try {
+    return bowser.parse(userAgent).browser;
+  } catch (error) {
+    logger.error(`Error parsing user agent "${JSON.stringify(userAgent)}": ${error.toString()}`);
+    return null;
+  }
+};
+
+module.exports = async () => {
+  const { rows } = await db.medicUsersMeta.query('users-meta/device_by_user', { group: true });
+  return rows.map(doc => {
+    const user = doc.key;
+    const date = doc.value.date;
+    const deviceId = doc.value.device.deviceId;
+    const browser = doc.value.device.userAgent && getBrowser(doc.value.device.userAgent);
+    const { apk, android, cht, settings } = doc.value.device.versions;
+    return {
+      user,
+      deviceId,
+      date,
+      browser: {
+        name: browser?.name || undefined,
+        version: browser?.version || undefined,
```

Comment:
Minor 
Do these sometimes have values like `null` or `false`? We can just omit the `|| undefined` and just let it be falsy.

Reply:
So, if there is a `userAgent` string, but `bowser` cannot parse it, then we get a `browser` object, but `name` and `version` are both empty strings.  I added `|| undefined` to try and unify the behavior for the cases were there is no `userAgent` and the `userAgent` string is not valid. Without  `|| undefined`, this code will return `browser: { name: undefined, version: undefined }` when there is no `userAgent` and `browser: { name: '', version: '' }` when the `userAgent` cannot be parsed.

Let me know if you think it is fine to just have different behavior (or if there is a way we could make this behave better)!